From: SeongJae Park <sj@kernel.org>
Date: Thu, 3 Oct 2024 15:20:36 -0700
Subject: [PATCH] selftests/damon/huge_count_read_write: provide sufficiently
 large buffer for DEPRECATED file read

The test binary crashes with segmentation fault while reading DEPRECATED file
of DAMON debugfs interface, since the file contains larger than the buffer size
content.  Increase the buffer size to avoid the segmentation fault, and make
the test output cleaner.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 tools/testing/selftests/damon/huge_count_read_write.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/damon/huge_count_read_write.c b/tools/testing/selftests/damon/huge_count_read_write.c
index a6fe0689f88d..f3c199dc8eba 100644
--- a/tools/testing/selftests/damon/huge_count_read_write.c
+++ b/tools/testing/selftests/damon/huge_count_read_write.c
@@ -18,7 +18,7 @@
 void write_read_with_huge_count(char *file)
 {
 	int filedesc = open(file, O_RDWR);
-	char buf[25];
+	char buf[256];
 	int ret;
 
 	printf("%s %s\n", __func__, file);
-- 
2.39.5

